main.go

```diff
@@ -110,11 +119,41 @@ func main() {
 	}
 
 	if !useTLS {
-		log.Info("serving gRPC-Gateway and OpenAPI Documentation on http://", gatewayAddr)
-		log.Fatalln(gwServer.ListenAndServe())
+		logger.Info("serving gRPC-Gateway and OpenAPI Documentation", zap.String("gatewayAddr", "http://"+gatewayAddr))
+		if err := gwServer.ListenAndServe(); err != nil {
+			logger.Fatal("failed to serve gRPC-Gateway", zap.Error(err), zap.Bool("tls", false))
+		}
 	}
 
 	gwServer.TLSConfig = tlsConfig
-	log.Info("serving gRPC-Gateway and OpenAPI Documentation on https://", gatewayAddr)
-	log.Fatalln(gwServer.ListenAndServeTLS("", ""))
+	logger.Info("serving gRPC-Gateway and OpenAPI Documentation", zap.String("gatewayAddr", "https://"+gatewayAddr))
+	if err := gwServer.ListenAndServeTLS("", ""); err != nil {
+		logger.Fatal("failed to serve gRPC-Gateway", zap.Error(err), zap.Bool("tls", true))
+	}
+}
+
+func initLogger() *zap.Logger {
+	logConf := zap.NewProductionEncoderConfig()
```

Comment:
That will enable JSON logging in development too, that might be too hard to read.

Feel free to adapt (but do not import) that code: https://github.com/percona-platform/saas/blob/main/pkg/logger/global.go

Reply:
As one of approach
ff96bba88cd3562dec5689eac70cc0afb60816ad